packages/components/src/components/form/component.tsx

```diff
@@ -72,6 +79,8 @@ export class KolForm implements FormAPI {
 		);
 	}
 
+	errorListElement!: HTMLElement;
```

Comment:
Vom Coding-sytle fände ich es sehr angenehm, wenn fields immer oben in der Klasse definiert werden und danach die Methoden und Funktionen. Und mit accessor wäre noch schön. :)